docs/how-to-guides/config-settings.rst

```diff
@@ -59,3 +59,63 @@ immediately install it, replace ``wheel`` with ``install``.  See the
 
 .. _build: https://pypa-build.readthedocs.io/en/stable/
 .. _pip: https://pip.pypa.io/
+
+
+Using a persistent build directory
+==================================
+
+By default, ``meson-python`` will create a temporary build directory, which
+gets cleaned up after the sdist or wheel has been built, or if the build fails.
+The ``build-dir`` config seting (see :ref:`reference-config-settings`) can be
+used to build in a user-specified build directory instead. For example:
+
+.. tab-set::
+
+    .. tab-item:: pypa/build
+        :sync: key_pypa_build
+
+	.. code-block:: console
+
+	   $ python -m build -Cbuild-dir=build
+
+    .. tab-item:: pip
+        :sync: key_pip
+
+	.. code-block:: console
+
+	   $ python -m pip install . -Cbuild-dir=build
+
+
+The ``build`` directory will now contain all the build output created by
```

Comment:
I would say "build artifacts and support files" instead than "build output". I think we use the latter to indicate the console logs usually.

Reply:
adopted